### DISCUSSION TITLE
[Feature] able to lock the mobile app

### DISCUSSION THREAD

**Discussion by EdeAbreu23** [immich-app/immich#11808] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add functionality to lock the app with pin/fingerprint

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Locking the whole app can be done with existing tools (first search result: https://www.airdroid.com/parent-control/lock-apps-android/).

As for more fine-grained locking, #2451.
